### ISSUE TITLE
Inventory Not Working Add And Remove Item Not Working

### ISSUE THREAD

**Issue by a-nonymou-s** [qbcore-framework/qb-inventory#336]
**Describe the bug**
nil triggered QBCore:Server:AddItem by ID 1 for 1 burger-softdrink. This event is deprecated due to exploitation, and will be removed soon. Adjust your events accordingly to do this server side with player functions.
I Don't receive any items on my inventory when i'm making a coffee or a burger for example
there's a notification that i've received a coffee but on my inventory there's nothing

**Screenshots**
![image](https://user-images.githubusercontent.com/72033787/186914182-71db7f7a-7c06-4ca8-bd6a-346e836de15c.png)

**Questions (please complete the following information):**
 - When you last updated: tomorrow morning
 - Are you using custom resource? which ones? qb-burgershot , mikel coffee
 - Have you renamed `qb-` to something custom? [e.g. yes/no] no

**Comment by nzkfc** [comment]
This is because all scripts that use the older QBCore:HasItem should now use QBCore.Functions.HasItem.

Therefore whatever scripts you use that have not been updated to account for this change across qbcore scripts will need to do so.

https://discord.com/channels/831626422232678481/883165186247254127/1012721593425854594

**Comment by LaserFeesh** [comment]
Is there a guide on how to fix this?  It's broken using items in my game. I have both qb-core and qb-inventory updated to the newest versions and it's still happening.

**Comment by nzkfc** [comment]
No, just have to wait for devs to fix or use any other inventory as none of
them have this issue.

I've raised this issue being fairly high priority in discord month or so
ago and was told if its an issue they would sort but seems it isn't.
There's a lot of issues piled up across qb-core so I guess they super busy
at the moment.

**Comment by LaserFeesh** [comment]
Thanks for the response. I'll look into some other inventories possibly.

**Comment by Maxhero54** [comment]
This event is deprecated due to exploitation, and will be removed soon. Adjust your events accordingly to do this server side with player functions.  same problem  any one know  how can solve ?
